### PR TITLE
add set meta data file

### PR DESCRIPTION
**Pull request by DevOlegKosarev**

### CODE REVIEW

**Comment by DevOlegKosarev** [comment]
add set meta data document

File: src/SimpleXLSXGen.php [hunk 1]
```diff
@@ -312,11 +341,21 @@ protected function _write($fh)
                 foreach ($this->sheets as $k => $v) {
                     $s .= '<sheet name="' . $this->esc($v['name']) . '" sheetId="' . ($k + 1) . '" r:id="rId' . ($k + 1) . '"/>';
                 }
-                $template = str_replace('{SHEETS}', $s, $template);
+                $search = ['{SHEETS}', '{APP}'];
+                $replace = [$s, $this->template];
```
Comment (shuchkin) [inline src/SimpleXLSXGen.php:345]: $this->application

File: README.md [hunk 2]
```diff
@@ -142,6 +142,17 @@ $xlsx->freezePanes('C3');
 // Column A is on the far right, Column B is one column left of Column A, and so on. Also, information in cells is displayed in the Right to Left format.
 $xlsx->rightToLeft();
 
+// Set Meta Data Files
+// this data in propertis Files and Info file in Office 
+$xlsx->setAuthor("Sergey Shuchkin")
```
Comment (shuchkin) [inline README.md:147]: use fluid interface and '
```php
$xlsx->setAuthor('Bill Gates')
    ->setCompany('Microsoft')
...
```

File: src/SimpleXLSXGen.php [hunk 3]
```diff
@@ -312,11 +341,21 @@ protected function _write($fh)
...
+                $template = str_replace($search, $replace, $template);
+                $this->_writeEntry($fh, $cdrec, $cfilename, $template);
+                $entries++;
+            } elseif ($cfilename === 'docProps/app.xml') {
+                $search = ['{APP}', '{COMPANY}', '{MANAGER}'];
+                $replace = [$this->application, $this->company, $this->manager];
```
Comment (shuchkin) [inline src/SimpleXLSXGen.php:351]: escape strings pls
```
$replace = [$this->esc($this->application), ....)
```

File: src/SimpleXLSXGen.php [hunk 4]
```diff
@@ -312,11 +341,21 @@ protected function _write($fh)
...
+                $template = str_replace($search, $replace, $template);
                 $this->_writeEntry($fh, $cdrec, $cfilename, $template);
                 $entries++;
             } elseif ($cfilename === 'docProps/core.xml') {
-                $template = str_replace('{DATE}', gmdate('Y-m-d\TH:i:s\Z'), $template);
+                $search = ['{DATE}', '{AUTHOR}', '{TITLE}', '{SUBJECT}', '{KEYWORD}', '{DESCRIPTION}', '{CATEGORY}'];
+                $replace = [gmdate('Y-m-d\TH:i:s\Z'), $this->author, $this->title, $this->subject, $this->keywords, $this->description, $this->category];
```
Comment (shuchkin) [inline src/SimpleXLSXGen.php:357]: escape author, title, subject, keywords, desc, cat

**Pull request closed without merging**